utils/eth1.go

```diff
@@ -196,15 +210,16 @@ func formatAddress(address []byte, token []byte, name string, isContract bool, l
 		ret = "<i class=\"fas fa-file-contract mr-1\"></i>" + ret
 	}
 
-	// not a link
-	if len(link) < 1 {
+	if len(link) == 0 {
+		// not a link
 		ret += fmt.Sprintf(`<span data-html="true" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="%s" data-container="body">%s</span>`, tooltip, name)
 	} else {
-		// link & token
 		if token != nil {
+			// link & token
 			ret += fmt.Sprintf(`<a href="/`+link+`/0x%x#erc20Txns" target="_parent" data-html="true" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="%s">%s</a>`, address, tooltip, name)
-		} else { // just link
-			ret += fmt.Sprintf(`<a href="/`+link+`/0x%x`+urlFragment+`" target="_parent" data-html="true" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="%s">%s</a>`, address, tooltip, name)
+		} else {
+			// just link
+			ret += fmt.Sprintf(`<a href="/`+link+`/0x%x" target="_parent" data-html="true" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="%s">%s</a>`, address, tooltip, name)
```

Comment:
**suggestion:** It's odd to add `link` to the text like this when we already have an `Sprintf`.
How about an `%s` instead?

Reply:
You are right, fixed.